dist/codecov_ats.sh

```diff
@@ -44,31 +44,31 @@ codecovcli_args=""
 pip install codecov-cli
 
 create_commit_args=""
-[[ -n $INPUTS_FAIL_ON_ERROR ]] && $command_args+="-Z "
-[[ -n $INPUTS_OVERRIDE_PARENT ]] && $command_args+="--parent-sha ${INPUTS_OVERRIDE_PARENT} "
-[[ -n $INPUTS_OVERRIDE_PR ]] && $command_args+="-P ${INPUTS_OVERRIDE_PR} "
-[[ -n $INPUTS_OVERRIDE_BRANCH ]] && $command_args+="-B ${INPUTS_OVERRIDE_BRANCH} "
-[[ -n $INPUTS_OVERRIDE_COMMIT ]] && $command_args+="-C ${INPUTS_OVERRIDE_COMMIT} "
-[[ -n $INPUTS_OVERRIDE_SLUG ]] && $command_args+="-r ${INPUTS_OVERRIDE_SLUG} "
+[[ -n $INPUTS_FAIL_ON_ERROR ]] && $create_commit_args+="-Z "
```

Comment:
shouldn't you remove the `$` symbol? so that it adds to the var instead of expanding it...

Reply:
Like you do on lines 39-41